config/initializers/panoptes.rb

```diff
@@ -0,0 +1,5 @@
+module Panoptes
+  def self.lifecycled_live_window
+    @lifecycled_live_window ||= (ENV["LIVE_WINDOW"] || 15)
```

Comment:
Actually, probably `Integer(ENV["LIVE_WINDOW"] || 15)` might be better than `to_i`, since it'll raise if it's not parseable.